projects/packages/forms/src/contact-form/class-contact-form-endpoint.php

```diff
@@ -34,6 +34,26 @@ public function register_routes() {
 			)
 		);
 
+		register_rest_route(
+			$this->namespace,
+			$this->rest_base . '/integration-status/(?P<slug>[\w-]+)',
+			array(
+				'methods'             => \WP_REST_Server::READABLE,
+				'callback'            => array( $this, 'get_integration_status' ),
+				'permission_callback' => array( $this, 'get_items_permissions_check' ),
+				'args'                => array(
+					'slug' => array(
+						'type'              => 'string',
+						'required'          => true,
+						'sanitize_callback' => 'sanitize_text_field',
```

Comment:
This should check for the supported slugs and return any included ones.

Reply:
We do check for matching slugs in the callback. Does we think we also need a check here?